Development/nmos/json_fields.h

```diff
@@ -238,6 +238,71 @@ namespace nmos
+            const web::json::field_as_string_or token_endpoint{ U("token_endpoint"),{} };
+            const web::json::field_as_string_or registration_endpoint{ U("registration_endpoint"),{} };
+            const web::json::field_as_array scopes_supported{ U("scopes_supported") };  // OPTIONAL
+            const web::json::field_as_array response_types_supported{ U("response_types_supported") };
+            const web::json::field_as_array response_modes_supported{ U("response_modes_supported") }; // OPTIONAL
+            const web::json::field_as_array grant_types_supported{ U("grant_types_supported") }; // OPTIONAL
+            const web::json::field_as_array token_endpoint_auth_methods_supported{ U("token_endpoint_auth_methods_supported") }; // OPTIONAL
+            const web::json::field_as_array token_endpoint_auth_signing_alg_values_supported{ U("token_endpoint_auth_signing_alg_values_supported") }; // OPTIONAL
+            const web::json::field_as_string service_documentation{ U("service_documentation") }; // OPTIONAL
+            const web::json::field_as_array ui_locales_supported{ U("ui_locales_supported") }; // OPTIONAL
+            const web::json::field_as_string op_policy_uri{ U("op_policy_uri") }; // OPTIONAL
+            const web::json::field_as_string op_tos_uri{ U("op_tos_uri") }; // OPTIONAL
+            const web::json::field_as_string revocation_endpoint{ U("revocation_endpoint") }; // OPTIONAL
+            const web::json::field_as_array revocation_endpoint_auth_methods_supported{ U("revocation_endpoint_auth_methods_supported") }; // OPTIONAL
+            const web::json::field_as_array revocation_endpoint_auth_signing_alg_values_supported{ U("revocation_endpoint_auth_signing_alg_values_supported") }; // OPTIONAL
+            const web::json::field_as_string introspection_endpoint{ U("introspection_endpoint") }; // OPTIONAL
+            const web::json::field_as_array introspection_endpoint_auth_methods_supported{ U("introspection_endpoint_auth_methods_supported") }; // OPTIONAL
+            const web::json::field_as_array introspection_endpoint_auth_signing_alg_values_supported{ U("introspection_endpoint_auth_signing_alg_values_supported") }; // OPTIONAL
+            const web::json::field_as_array code_challenge_methods_supported{ U("code_challenge_methods_supported") };
+
+            // Client Metadata
+            const web::json::field_as_value client_metadata{ U("client_metadata") };
+            // see https://tools.ietf.org/html/rfc7591#section-2
+            // see https://tools.ietf.org/html/rfc7591#section-3.1
+            // see https://tools.ietf.org/html/rfc7591#section-3.2
+            const web::json::field_as_array redirect_uris{ U("redirect_uris") };
+            //const web::json::field_as_string token_endpoint_auth_method{ U("token_endpoint_auth_method") }; // OPTIONAL already defined in settings
```

Comment:
Commented out - should this be removed?

Reply:
This is a good identification showing why the `token_endpoint_auth_method` is not defined in this file, as it is already defined in settings (settings.h)